nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -34,8 +37,42 @@
 
 __all__ = [
     'AudioToCharDALIDataset',
+    'AudioToBPEDALIDataset',
 ]
 
+__DALI_MINIMUM_VERSION__ = "1.4"
```

Comment:
I would add a comment why this is the minimal DALI version.

Reply:
Done